Go 1.15: conversion from untyped int to string yields a string of one rune

Go 1.15 beta 1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/gomemcache-a41fca850d0b6f392931a78cbae438803ea0b886/_build/src
         PATH: /builddir/build/BUILD/gomemcache-a41fca850d0b6f392931a78cbae438803ea0b886/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/gomemcache-a41fca850d0b6f392931a78cbae438803ea0b886/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/bradfitz/gomemcache/version=0 -X github.com/bradfitz/gomemcache/version.commit=a41fca850d0b6f392931a78cbae438803ea0b886 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/bradfitz/gomemcache
github.com/bradfitz/gomemcache/memcache
# github.com/bradfitz/gomemcache/memcache
./memcache_test.go:130:33: conversion from untyped int to string yields a string of one rune
FAIL	github.com/bradfitz/gomemcache/memcache [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/memcache/memcache_test.go
+++ b/memcache/memcache_test.go
@@ -127,7 +127,7 @@ func testWithClient(t *testing.T, c *Client) {
 	if err != ErrMalformedKey {
 		t.Errorf("set(foo bar) should return ErrMalformedKey instead of %v", err)
 	}
-	malFormed = &Item{Key: "foo" + string(0x7f), Value: []byte("foobarval")}
+	malFormed = &Item{Key: "foo" + string(rune(0x7f)), Value: []byte("foobarval")}
 	err = c.Set(malFormed)
 	if err != ErrMalformedKey {
 		t.Errorf("set(foo<0x7f>) should return ErrMalformedKey instead of %v", err)

